File: 461.c
int hammingDistance(int x, int y) {
    int temp;
    if(x<y)
    {
        temp = x;
        x = y;
        y = temp;
    }
    temp = 0;
    while(x)
    {
        if(x%2!=y%2)
            temp++;
        x/=2,y/=2;
    }
    return temp;
}

File: Python/461.py
class Solution():
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        return bin(x ^ y).count("1")

ans = Solution()
x= ans.hammingDistance(1, 4)
print(x)
